### PR TITLE
2019/kw15/design_differences_post

### PR DESCRIPTION
**Pull request by Tirokk**
> [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-08T13:56:03Z" title="Monday, April 8th 2019, 3:56:03 pm +02:00">Apr 8, 2019</time>_
_Merged <time datetime="2019-04-12T12:14:24Z" title="Friday, April 12th 2019, 2:14:24 pm +02:00">Apr 12, 2019</time>_
---

## Pullrequest

image

fixed design differences between alpha & nitro - Post Page
- [X] Title and Author Position reversed
- [X] Design & Format of Post-Date + Position
- [X] No Category Names
- [X] Actions/Reactions & Tags reversed

### Issues

- [X] fixes https://github.com/Human-Connection/Human-Connection/issues/392

### Checklist

- [X] None

### How2Test

- [X] None

### Todo

- [X] None

### CODE REVIEW

**Comment by Tirokk** [comment]
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-10T17:47:38Z" title="Wednesday, April 10th 2019, 7:47:38 pm +02:00">Apr 10, 2019</time>_

---


@ulfgebhardt I think the build server is failing for good reason, could you check?

**Comment by Tirokk** [comment]
> [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-04-11T07:22:10Z" title="Thursday, April 11th 2019, 9:22:10 am +02:00">Apr 11, 2019</time>_

---


Moin @roschaefer , the Jest component test of `RelativeDateTime` works fine and looks nice. Congrats !!!

File: webapp/components/RelativeDateTime/index.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import formatRelative from 'date-fns/formatRelative'
+import { enUS, de, nl, fr, pt, es /*, pl*/ } from 'date-fns/locale'
+const locales = {
+  en: enUS,
+  de,
+  nl,
+  fr,
+  es,
+  pt
+  // pl
```
Comment (Tirokk) [inline webapp/components/RelativeDateTime/index.vue:15]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-12T08:57:12Z" title="Friday, April 12th 2019, 10:57:12 am +02:00">Apr 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,33 @@
+<template>
+  <span>{{ relativeDateTime }}</span>
+</template>
+
+<script>
+import formatRelative from 'date-fns/formatRelative'
+import { enUS, de, nl, fr, pt, es /*, pl*/ } from 'date-fns/locale'
+const locales = {
+  en: enUS,
+  de,
+  nl,
+  fr,
+  es,
+  pt
+  //pl
```


What's wrong with Polish? :thinking: 

@ulfgebhardt by the way, I think I missed one test case: What happens, if you have a `i18n.locale()` but this locale is not in the list above? Could you create a test case for it? I would like to have a default in that case.

**Comment by Tirokk** [comment]
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-12T09:00:35Z" title="Friday, April 12th 2019, 11:00:35 am +02:00">Apr 12, 2019</time>_

---


@ulfgebhardt the build server fails at `✖ 06.WritePost.feature` on both builds. I don't think this is a false positive (test fails, but implementation is fine) but a true positive (test fails because implementation is broken). Could you run the cypress tests locally and have a check?

File: webapp/components/RelativeDateTime/index.vue (hunk 1 above)
Comment (Tirokk) [reply webapp/components/RelativeDateTime/index.vue:15]: > [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-12T09:17:14Z" title="Friday, April 12th 2019, 11:17:14 am +02:00">Apr 12, 2019</time>_

---


Polish is currently not implemented on library side.

**Comment by Tirokk** [comment]
> [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-12T11:33:47Z" title="Friday, April 12th 2019, 1:33:47 pm +02:00">Apr 12, 2019</time>_

---


image
image
image
image

Cypress tests are not working with chrome or electron
```
yarn cypress:setup
yarn cypress:run (changed browser)
```

Error message from travis is not reproducable
image
image

Posts can be created - works as expected

**Review by Mogge — APPROVED** [review]
@ulfgebhardt I can only repeat what I said: Writing tests will uncover design flaws, highlights dependencies and forces you to refactor hard-to-maintain code.

Together with @Tirokk it took us some hours to find the appropriate solution to a design flaw with our custom plugin `vue-filters`: Don't use it.

**Pull request closed without merging**